Dart/src/com/jetbrains/lang/dart/ide/refactoring/DartRenameHandler.java

```diff
@@ -55,14 +65,7 @@ public void invoke(@NotNull Project project, @NotNull PsiElement[] elements, Dat
 
   @Override
   public boolean isAvailableOnDataContext(DataContext dataContext) {
-    final PsiElement element = CommonDataKeys.PSI_ELEMENT.getData(dataContext);
-    if (element instanceof DartNamedElement) {
-      return true;
-    }
-    if (element instanceof DartComponent) {
-      return true;
-    }
-    return false;
+    return true;
```

Comment:
This extension point doesn't support `lang` attribute in `plugin.xml`, so this `return true` breaks renaming in all languages in all JetBrains IDEs. I'll fix.